basic_nodes/src/main/java/club/mondaylunch/gatos/basicnodes/ListSortNodeType.java

```diff
@@ -0,0 +1,57 @@
+package club.mondaylunch.gatos.basicnodes;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import club.mondaylunch.gatos.core.data.DataBox;
+import club.mondaylunch.gatos.core.data.DataType;
+import club.mondaylunch.gatos.core.data.ListDataType;
+import club.mondaylunch.gatos.core.graph.connector.NodeConnector;
+import club.mondaylunch.gatos.core.graph.type.NodeType;
+
+public class ListSortNodeType extends NodeType.Process {
+    @Override
+    public Map<String, DataBox<?>> settings() {
+        return Map.of();
+    }
+
+    @Override
+    public Set<NodeConnector.Input<?>> inputs(UUID nodeId, Map<String, DataBox<?>> settings, Map<String, DataType<?>> inputTypes) {
+        return Set.of(
+            new NodeConnector.Input<>(nodeId, "input", ListDataType.GENERIC_LIST));
+    }
+
+    @Override
+    public Set<NodeConnector.Output<?>> outputs(UUID nodeId, Map<String, DataBox<?>> settings, Map<String, DataType<?>> inputTypes) {
+        var outputType = inputTypes.get("input");
+        return Set.of(
+            new NodeConnector.Output<>(nodeId, "output", outputType));
+    }
+
+    @Override
+    @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
I don't think there are any rawtypes here to suppress?

Reply:
The cast to List<Comparable> is being suppressed here

Reply:
yeah, I see the unchecked, I'm talking about the rawtypes suppression

Reply:
so the cast to Comparable throws a rawtypes error since it's Comparable<T>, but i couldn't find a way to make it work with either ? or T as the type parameter so yeah